wolox-training/vs-rails
# Basic blog with articles and comments

## Basic blog with articles and comments

**Summary**
- The blog lists all the articles, you can create, edit or delete an article.
- You can see a specific article and its comments
- You can create new comments in articles, and also destroy them
- Some actions such as edit, delete, or create will require a basic authentication, which for now is done with the **http_basic_authenticate_with** method.

**Test plan**
- Try creating a new article, making comments in it, editing it, deleting it.
- Try destroying comments
- See the list of articles, and the list of comments in every article
- When the blog asks for an user and password, the credentials are *vale* and *secret*

**Trello card**
https://trello.com/c/zothD80A/22-crear-blog-base
